web/src/component/CountDownInput.js

```diff
@@ -72,55 +82,113 @@ export const CountDownInput = (props) => {
       } else {
         Setting.showMessage("error", i18next.t("signup:Unknown Check Type"));
       }
-    })
-  }
+    });
+  };
 
   const renderCaptcha = () => {
     return (
-      <Col>
-        <Row
-          style={{
-            backgroundImage: `url('data:image/png;base64,${captchaImg}')`,
-            backgroundRepeat: "no-repeat",
-            height: "80px",
-            width: "200px",
-            borderRadius: "3px",
-            border: "1px solid #ccc",
-            marginBottom: 10
-          }}
-        />
-        <Row>
-          <Input placeholder={i18next.t("general:Enter the code")} onChange={e => setKey(e.target.value)} />
-        </Row>
-      </Col>
-    )
-  }
+        <Col>
+          <Row
+              style={{
+                backgroundImage: `url('data:image/png;base64,${captchaImg}')`,
+                backgroundRepeat: "no-repeat",
+                height: "80px",
+                width: "200px",
+                borderRadius: "3px",
+                border: "1px solid #ccc",
+                marginBottom: 10,
+              }}
+          />
+          <Row>
+            <Input
+                placeholder={i18next.t("general:Enter the code")}
+                onChange={(e) => setKey(e.target.value)}
+            />
+          </Row>
+        </Col>
+    );
+  };
 
   const renderCheck = () => {
     if (checkType === "captcha") return renderCaptcha();
     return null;
+  };
+
+  if (prefix !== null) {
+    switch (prefix) {
+      case "VerifiedOutlined":
+        return (
+            <Input
+                addonBefore={textBefore}
+                placeholder={placeHolder}
+                prefix={<VerifiedOutlined />}
+                onChange={(e) => onChange(e.target.value)}
+                addonAfter={
+                  <div>
+                    <button
+                        onClick={clickButton}
+                        style={{ backgroundColor: "#fafafa", border: "none" }}
+                    >
+                      {buttonText}
+                    </button>
+                    <Modal
+                        visible={visible}
+                        onCancel={handleCancel}
+                        onOk={handleOk}
+                    >
+                      {renderCheck()}
+                    </Modal>
+                  </div>
+                }
+            />
+        );
```

Comment:
These lines repeated for 3 times, but only 1 line of them are different. I suggest we can write these lines in a more elegant way.